core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2Authenticator.java

```diff
@@ -14,6 +14,7 @@
 package io.trino.server.security.oauth2;
```

Comment:
There is a typo in the commit message.

> user couldn't log in by using standard jwt token.

Can you please explain why in the commit message too?